### ISSUE TITLE
PG Migrations using Schemas

### ISSUE THREAD

**Issue by twonil-jordan-stout** [drizzle-team/drizzle-kit-mirror#49]
I noticed a few things when running migrations when everything is within a schema.

Foreign keys are missing the schema.  The output below, "user_roles" is in a schema "users":
```sql
DO $$ BEGIN
 ALTER TABLE "user_roles" ADD CONSTRAINT "user_roles_user_id_users_id_fk" FOREIGN KEY ("user_id") REFERENCES "users"("id") ON DELETE cascade ON UPDATE cascade;
EXCEPTION
 WHEN duplicate_object THEN null;
END $$;
```

Same with creating indexes. "users" should be "users"."users":

```sql
CREATE UNIQUE INDEX IF NOT EXISTS "username_idx" ON "users" ("username");
```

**Comment by AndriiSherman** [comment]
We have a few issues pointing that out
Fixing schemas is a high priority for us

**Comment by AndriiSherman** [comment]
Fixed in 0.19.0
